multiplexer/abci/remote_v1.go

```diff
@@ -0,0 +1,636 @@
+package abci
+
+import (
+	"context"
+	"math"
+
+	"google.golang.org/grpc"
+
+	abciv2 "github.com/cometbft/cometbft/abci/types"
+	cryptov2 "github.com/cometbft/cometbft/proto/tendermint/crypto"
+	typesv2 "github.com/cometbft/cometbft/proto/tendermint/types"
+	abciv1 "github.com/tendermint/tendermint/abci/types"
+	cryptov1 "github.com/tendermint/tendermint/proto/tendermint/crypto"
+	typesv1 "github.com/tendermint/tendermint/proto/tendermint/types"
+	versionv1 "github.com/tendermint/tendermint/proto/tendermint/version"
```

Comment:
[random comment]
the switch to cometbft in the go mod vs using tendermint actually worked out so that we can import two different versions that aren't v1 v2 etc

Reply:
definitely! we would have had to copy the protos otherwise